src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyHostBuilder.cs

```diff
@@ -47,7 +47,7 @@ public static WebAssemblyHostBuilder CreateDefault(string[] args = default)
         /// <summary>
         /// Creates an instance of <see cref="WebAssemblyHostBuilder"/> with the minimal configuration.
         /// </summary>
-        internal WebAssemblyHostBuilder(WebAssemblyJSRuntimeInvoker jsRuntimeInvoker)
+        internal WebAssemblyHostBuilder(IJSUnmarshalledRuntime jsRuntime)
```

Comment:
@SteveSandersonMS I used this as an opportunity to remove `WebAssemblyJSRuntimeInvoker`. The comment block here: https://github.com/dotnet/aspnetcore/blob/master/src/Components/WebAssembly/WebAssembly/src/Services/WebAssemblyJSRuntimeInvoker.cs#L9-L18 no longer applies now that we recommend using the interface as the primary mechanism to work with unmarshalled data. 

Let me know if you disagree.

Reply:
Yep, very good.

For our internal logic it would be (marginally) better to use the concrete `DefaultWebAssemblyJSRuntime.Instance` directly wherever possible to avoid virtual dispatch and only use `IJSUnmarshalledRuntime` in cases where we need a unit testing seam (i.e., places we used to use `WebAssemblyJSRuntimeInvoker`). What you've done here fits into that category so the use of `IJSUnmarshalledRuntime` is totally fine here.